tools/publisher/src/subcommand/yank_release.rs

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::cargo;
+use anyhow::{bail, Context, Result};
+use clap::Parser;
+use dialoguer::Confirm;
+use regex::Regex;
+use smithy_rs_tool_common::shell::ShellOperation;
+use smithy_rs_tool_common::versions_manifest::{Release, VersionsManifest};
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+use std::sync::Arc;
+use tokio::sync::Semaphore;
+use tracing::info;
+
+const MAX_CONCURRENCY: usize = 5;
+
+#[derive(Parser, Debug)]
+pub struct YankReleaseArgs {
+    /// The aws-sdk-rust release tag to yank
```

Comment:
docs should say something like: "this will download `versions.toml` from GitHub at the tagged release and yank crates that had that version" or something